doc/Building Tribler on Windows.md

```diff
@@ -0,0 +1,43 @@
+This page contains information about building Tribler on Windows. The final result you should have is a `.exe` file which, when opened, enables users to install Tribler on their system. 
+This guide installs a 64-bit version of Tribler and has been tested on Windows 10 and Windows 2008 Server R2, 64-bit. It is recommended to run this builder on a system that is already able to run Tribler without problems (it means that all the required packages required by Tribler are installed already).  In case you want to build a 32 bit version, just install all the dependencies mentioned here in 32 bit version.
+Information about setting up a developer environment on Windows can be found [here](https://github.com/Tribler/tribler/blob/devel/doc/Tribler%20development%20on%20Windows.md).
+
+**When you have installed zope, an empty ``__init__.py`` file must be present in the zope folder. If this file is missing, a ``No module named zope`` error will be thrown. Create this file in the ``site-packes/zope`` folder if it does not exist.**
+ 
+# Required packages
+To build a Tribler installer, you'll need some additional scripts and packages:
+* The git command tools are required to fetch the latest release information. These can be downloaded from [here](https://git-scm.com/download/win). 
+* Py2Exe, a tool to create an executeable from python files. Grab the latest version [here](http://sourceforge.net/projects/py2exe/files/py2exe/).
+* The builder needs to find all packages that are required by Tribler so make sure you can run Tribler on your machine and that there are no missing dependencies.
+* Nullsoft Scriptable Install System (NSIS) is a script-driven Installer authoring tool for Microsoft Windows with minimal overhead. It can be downloaded [here](http://nsis.sourceforge.net/Download).
+* Three plugins are required.The UAC plugin is the first. This can be downloaded from [here](http://nsis.sourceforge.net/UAC_plug-in). How to install a plugin can be found [here](http://nsis.sourceforge.net/How_can_I_install_a_plugin).
+* The second plugin that is needed is AccessControl plug-in. It can be downloaded [here](http://nsis.sourceforge.net/AccessControl_plug-in).
+* The third plugin required is NSIS Simple Firewall Plugin. You can download it [here](http://nsis.sourceforge.net/NSIS_Simple_Firewall_Plugin).
+* A version of Microsoft Visual Studio should be installed (we use 2012), but make sure you do not have the build-tools only. The full (community) edition can be downloaded [here](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx).
```

Comment:
We got missing dlls errors when we ran 2015, you can of course install these manually. With 2012 which we chose to install this time it may not be needed. We could test this, but for people on windows 8.1 or up, it is much easier to install VS then use dependency walker and find + install missing dlls.